dist/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+	<title>WeatherApp-E</title>
+</head>
+<body class="#e0f2f1 teal lighten-5">
+
+	<div id="search">
+		<input type="text" placeholder="enter city name" id="cityInp" class="#fafafa grey lighten-5">
+		<button id="inpBtn" class="btn waves-effect waves-light" type="submit" name="action">
+		 Check Weather
+		</button>
+	</div>
+
+	<div id="result-container" class="row">
+		
+	</div>
+
+	<script id="city-template" type="text/x-handlebars-template">
+		
+			<div id="placeholder" class="col s12 m6 l3 #e3f2fd blue lighten-5">
+				<h4 class="center-align">
+					{{name}}
+				</h4>
+
+				<div class="center-align content">
+					Temp: {{temperature}} <i class="fas fa-thermometer-half"></i>
+				</div>
+
+				<div class="center-align content">
+					Humidity: {{humidity}}%
+				</div>
+
+				<div class="center-align content">
+					<i class="fas fa-cloud-sun"></i> {{description}} 
```

Comment:
Looks like this will always be the same icon. You should change the icon based on the description. Actually the API gives you a way to do this.